crypto/evp/evp_lib.c

```diff
@@ -413,8 +413,10 @@ int EVP_Cipher(EVP_CIPHER_CTX *ctx, unsigned char *out,
          */
         int ret = -1;
         size_t outl = 0;
-        size_t blocksize = EVP_CIPHER_CTX_get_block_size(ctx);
+        int blocksize = EVP_CIPHER_CTX_get_block_size(ctx);
```

Comment:
This is OK, however below in those ccipher/cupdate/cfinal calls you need to cast blocksize to size_t explicitly before adding inl as otherwise there can be an overflow. Rather theoretical one but...